multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/Messages.java

```diff
@@ -673,6 +673,11 @@ public class Messages {
     public static final String GETTING_HISTORIC_VARIABLES_FOR_OPERATIONS_STARTED_AFTER_0 = "Getting historic variables for operations started after {0}";
     public static final String NO_ORPHANED_FILES_TO_DELETE = "No orphaned files to delete.";
     public static final String DELETING_ORPHANED_FILES_0 = "Deleting {0} orphaned files: {1}";
+    public static final String TAGS = "tags";
```

Comment:
The names of these constants are not really descriptive, but rather only repeat the same words that they contain. Maybe rename to something more reasonable, like 'SERVICE_ATTRIBUTE_TAGS" etc. Or better yet just don't make these service parameters into different constants and have separate custom error messages (if it makes sense). For example, 'service tags' are already used in log messages without them being separated into constants (better to follow established use of messages):
    public static final String UPDATING_SERVICE_TAGS = "Updating service tags for service \"{0}\"...";
    public static final String SERVICE_TAGS_UPDATED = "Service tags for service \"{0}\" updated";